prologin/concours/webide/rules.py

```diff
@@ -0,0 +1,50 @@
+from . import models
+from django.core.exceptions import ObjectDoesNotExist
+from django.contrib.auth import get_user_model
+
+import requests
+
+class WorkspaceVoucher:
```

Comment:
Add documentation, it's not trivial what are the dependencies for this class, and what it does.